More detailed data backup explanation (#41)
CatimaLoyalty/Website: _pages/faq.md

# Frequently asked questions

## Data

### Does Catima collect any data?

No. Catima does not collect any data and is designed to work fully offline. See also our [Privacy Policy](/privacy-policy).

### Can I back up and restore my data?

Catima supports both manual backups (through Catima's built-in Import/Export functionality) and automatic backups through Android Auto Backup.

> To back up or restore Catima's data manually, click the ⋮ (menu) button and choose Import/Export. Here you can export your data into a (password protected if wanted) .zip file and later restore it.

Reviewer: ```suggestion
To back up or restore Catima's data manually, click the ⋮ (menu) button and choose Import/Export. Here you can export your data into a .zip file and later restore it. The .zip file can be protected with a password for extra security.
```

Also: might look nicer to wrap `⋮` in a `<kbd>` tag or something?

Author: No strong opinion, which do you prefer?

With:
![image](https://github.com/CatimaLoyalty/Website/assets/1885159/dc5f847b-e676-469f-a802-62edf4fef68a)

Without:
![image](https://github.com/CatimaLoyalty/Website/assets/1885159/caad56ad-c5e0-494e-97ab-381da2942b91)

Feel free to edit your suggestion to include <kbd> if you prefer that version.

Reviewer: I prefer the `<kbd>`. I use that in the help for Jiten myself (though it's styled a bit differently by my CSS):

![image](https://github.com/CatimaLoyalty/Website/assets/1260687/fdc9cd99-d9a9-4726-a3f2-bae56c59e641)

Could always change the CSS as well if you like.

> When Android Auto Backup is enabled (it is enabled by default on most Android systems) [Google will automatically make encrypted backups of apps on their servers](https://support.google.com/android/answer/2819582). On custom ROMs like LineageOS, CalyxOS, etc., backups are generally handled by [SeedVault](https://calyxinstitute.org/projects/seedvault-encrypted-backup-for-android) instead. However, for reliability reasons, it is recommended to use Catima's built-in Import/Export functionality instead as this functionality is better tested.

Reviewer: ```suggestion
When Android Auto Backup is enabled (which it is by default on most Android systems) [Google will automatically make encrypted backups of apps on their servers](https://support.google.com/android/answer/2819582). On custom ROMs like LineageOS, CalyxOS, etc., backups are generally handled by [SeedVault](https://calyxinstitute.org/projects/seedvault-encrypted-backup-for-android) instead. However, for reliability reasons, it is recommended to use Catima's built-in Import/Export functionality instead as this functionality is better tested.
```
